radixdlt/src/test/java/com/radixdlt/sanitytestsuite/scenario/SanityTestScenarioRunner.java

```diff
@@ -0,0 +1,69 @@
+package com.radixdlt.sanitytestsuite.scenario;
+
+import com.google.gson.reflect.TypeToken;
+import com.radixdlt.sanitytestsuite.SanityTestSuiteTestLoader;
+import com.radixdlt.sanitytestsuite.model.SanityTestSuiteRoot;
+import com.radixdlt.sanitytestsuite.model.SanityTestVector;
+import com.radixdlt.sanitytestsuite.model.UnknownTestVector;
+import com.radixdlt.utils.JSONFormatter;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+import static org.junit.Assert.assertEquals;
+
+public abstract class SanityTestScenarioRunner<Vector extends SanityTestVector> {
```

Comment:
`Vector` is a standard, but deprecated Java class, so it's use here is very confusing.

Coding standards also generally use 1 or 2 uppercase letters for type arguments, otherwise they often get confused with class names.